index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1 id="title">
+ Coding Survey form
+ </h1>
+
+<p id="description">
+Coding practise on various programming languages. 
+</p>
```

Comment:
Please consider using one line of code instead of 3 different lines of code. It makes your code look clean.